client-core/src/wallet.rs

```diff
@@ -54,7 +54,10 @@ pub struct WalletRequest {
 }
 
 /// Interface for a generic wallet
-pub trait WalletClient: Send + Sync {
+pub trait WalletClient: Send + Sync + Clone {
+    /// recover address beforehand
+    /// ret: true== refetch transfer addresses
```

Comment:
What does this comment mean?

Reply:
if return value is true, refetch transfer address
i'll change

Reply:
because it's modifying addresses, changed name to
recover_addresses